src/components/Activities/Activity3/Game/TimeAllocation.tsx

```diff
@@ -117,35 +117,32 @@ function TimeAllocation(props: TimeAllocationProps): JSX.Element {
     setTutorialStage(tutorialStage+1);
   };
 
-  return <div id={'time-container'} className={tutorialStage < TUTORIAL_END ? 'enable-blur' : ''}>
-    <div id={'time-tutorial-overlay'} style={{display: `${tutorialStage >= TUTORIAL_END ? 'none' : ''}`}}/>
+  return <div id={'time-container'} className={isTutorial ? 'enable-blur' : ''}>
     {
-      Object.values([0, 1, 2]).map((index) => {
-        return (
-          <div key={index} id={'time-tutorial-bubble'} className={tutorialStage === index ? 'disable-blur' : ''}
-            style={{
-              display: `${(tutorialStage === index) ? '' : 'none'}`,
-              alignSelf: 'flex-center',
-            }}>
-            {displayTutorialText()}
-            <button className='playnet-button' style={{zIndex: 50}} onClick={advanceTutorial}>Continue</button>
-          </div>
-        );
-      })
+      isTutorial && <div id={'time-tutorial-overlay'} className={`${tutorialStyles[tutorialStage]} disable-blur`}>
+        <div id={'time-tutorial-bubble'}
+          className={'disable-blur'}>
+          {displayTutorialText()}
+          <button className='playnet-button' style={{zIndex: 50}} onClick={advanceTutorial}>Continue</button>
+        </div>
+      </div>
     }
     <div>
       Choose how much time to spend on each part of your project.
       <br/>
       (We recommend {LOW_DAY_THRESHOLD} - {HIGH_DAY_THRESHOLD} days total!)
     </div>
-    <div id={'options-grid'}>
-      {displayOptionIcons()}
+    <div id={'options-grid'} className={'disable-blur'}>
       {daysAllocation.map((curAlloc : number, index : number) => {
         const usableDays = daysLeft ? (daysLeft - sumDaysUsed() + curAlloc) : 0;
         return (
-          <div key={index} className={'centered-box'}>
-            <NumberSelection daysLeft={usableDays} itemType={index}
-              daysAllocation={daysAllocation} setDaysAllocation={setDaysAllocation} showWarning={isShowWarning()}/> days
+          <div key={index}
+            className={`option-container ${isTutorial && (tutorialStage === index ? 'disable-blur highlight-border' : 'enable-blur')}`}>
+            {optionIcons[index]}
```

Comment:
we can do this instead of optionIcons!
```suggestion
          <div key={DISPLAY_OPTIONS[index].text}
              className={'centered-box'}>
              <img src={DISPLAY_OPTIONS[index].src} />
              {DISPLAY_OPTIONS[index].text}
            </div>```